catalog/app/components/Pagination/index.js

```diff
@@ -0,0 +1,181 @@
+import assert from 'assert';
+import React, { PropTypes as PT } from 'react';
+import {
+  compose,
+  defaultProps,
+  pure,
+  setDisplayName,
+  setPropTypes,
+  withHandlers,
+  withProps,
+} from 'recompose';
+import styled from 'styled-components';
+import { FormattedMessage as FM } from 'react-intl';
+import DropDownMenu from 'material-ui/DropDownMenu';
+import MenuItem from 'material-ui/MenuItem';
+import IconButton from 'material-ui/IconButton';
+import { grey500 } from 'material-ui/styles/colors';
+
+import MIcon from 'components/MIcon';
+
+import { DEFAULT_PER_PAGE, DEFAULT_PER_PAGE_OPTIONS } from './constants';
+import messages from './messages';
+
+
+const UNIT_SIZE = '24px';
```

Comment:
minor, why two lines above and below this const?

Reply:
I'm used to separate major parts of file (imports, constants, logical groups of definitions, classes) with double newlines. I can remove the extra newlines if it's preferred.